fsm/handler.go

```diff
@@ -125,11 +125,11 @@ func HandleStateN1N2TransferPendingEventN1N2Transfer(event SmEvent, eventData *S
 	smCtxt := txn.Ctxt.(*smf_context.SMContext)
 
 	if err := producer.SendPduSessN1N2Transfer(smCtxt, true); err != nil {
-		stats.PublishMsgEvent(mi.Smf_msg_type_pdu_sess_update_req_failure)
+		stats.PublishMsgEvent(mi.Smf_msg_type_pdu_sess_modify_rsp_failure)
```

Comment:
req failure was correct right? If we are changign the statistics then we need to make sure we are consistent at all the state.

Reply:
The name of them was change in [Metricfunc PR 9](https://github.com/omec-project/metricfunc/pull/9/files#) but this was not actually changed in the SMF because SMF was using `Metricfunc v1.1.1` and these changes were made after that release tag. So, now that we are moving to a newer `Metricfunc tag`, we see these updates
```diff
...
...
-	Smf_msg_type_pdu_sess_create_req_failure
-	Smf_msg_type_pdu_sess_update_req
-	Smf_msg_type_pdu_sess_update_rsp_success
-	Smf_msg_type_pdu_sess_update_req_failure
+	Smf_msg_type_pdu_sess_create_rsp_failure
+	Smf_msg_type_pdu_sess_modify_req
+	Smf_msg_type_pdu_sess_modify_rsp_success
+	Smf_msg_type_pdu_sess_modify_rsp_failure
...
...
```